libsentrykube/quickpatch.py

```diff
@@ -135,16 +155,13 @@ def apply_patch(
 
     # Load the patch
     patch_data = yaml.safe_load(patch_data_str)
-    patches = patch_data.get("patches")
-    json_patch = jsonpatch.JsonPatch(patches)
+    patches = patch_data.get("patches", [])
```

Comment:
I think you should remove the regex complexity entirely and make the variable support a first class concept of your patch language. If you are parsing the path anyway to apply the patch you can process paths that contain `<asdasd>` as variables and substitute the variable value directly when applying the patch rather than doing a first pass with a regex.

Reply:
I think running a first pass is better, since this way we can support variables in more places than just the path or value fields

Reply:
Also, I feel like patch_json should just serve to apply the json changes to the resource json, and not mess around with <variables> (from the perspective of code modularity)

Reply:
Was discussed offline in Slack - conclusion: Would be more messy to include variable replacement logic in `patch_json` due to having to cover more cases. So will not change this